src/utils/error.js

```diff
@@ -2,49 +2,123 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-class UserInputError extends Error {
+import { getMessage } from './fluent.js'
+
+const genericErrorMessage = 'Something went wrong. Please try again or come back later.'
+const permissionErrorMessage = 'You don’t have permission to access this content.'
+const tooManyRequestsErrorMessage = 'Too many requests received. Please try again later. '
```

Comment:
Are these messages supposed to be localized? ie a part of a fluent file?

Reply:
We could localize them, but we currently don’t show them anywhere user-facing. It might make sense to only localize them as soon we surface them in case we need them to change.

Reply:
My preference would also be to use a consistent language for error messages internally (this will be sent to Sentry or the console), and localize as late in the process as possible, like when we format the final output for users.